x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -7,13 +7,18 @@
 import { SearchParams } from 'elasticsearch';
 import {
   PROCESSOR_EVENT,
-  TRACE_ID
+  TRACE_ID,
+  PARENT_ID,
+  TRANSACTION_DURATION,
+  SPAN_DURATION
 } from '../../../common/elasticsearch_fieldnames';
 import { Span } from '../../../typings/es_schemas/ui/Span';
 import { Transaction } from '../../../typings/es_schemas/ui/Transaction';
 import { rangeFilter } from '../helpers/range_filter';
 import { Setup } from '../helpers/setup_request';
 
+const MAX_ITEMS_RETURNED = 1000;
```

Comment:
Can you make this configurable via kibana.yml?

Reply:
created `apm_oss.maxTraceItems` to control this value.